src/wasm-lib/kcl/Cargo.toml

```diff
@@ -50,6 +50,9 @@ engine = []
 panic = "abort"
 debug = true
 
+[profile.bench]
+debug = true
```

Comment:
Flamegraphs of the benchmarks need this for accurately knowing the names of functions